Sources/Crypto/Keys/EC/Ed25519.swift

```diff
@@ -75,6 +75,18 @@ extension Curve25519 {
             var keyBytes: [UInt8] {
                 return self.baseKey.keyBytes
             }
+			
+			private func withUnsafeBytes<R>(_ body: (UnsafeRawBufferPointer) throws -> R) rethrows -> R {
+				return try self.baseKey.keyBytes.withUnsafeBytes(body)
+			}
+			
+			public static func ==(lhs: Self, rhs: Self) -> Bool {
```

Comment:
Part of #173 but I was unable to create a diff between just this PR and #173 since my branch is in a fork, otherwise I would have created this PR with the branch `curve25519_publickeys_equatable` as the target branch, so that we could have seen only the relevant diff.